inductiva/molecules/__init__.py

```diff
@@ -1,3 +1,4 @@
 #pylint: disable=missing-module-docstring
-from .scenarios import ProteinSolvation
-from .scenarios import MDWaterBox
+from .protein_solvation import ProteinSolvation, ProteinSolvationOutput
```

Comment:
This is also a weird import...
How would the code look like in this case?

Reply:
The point of this is for us to be able to do:
`inductiva.molecules.ProteinSolvation` instead of `inductiva.molecules.protein_solvation.ProteinSolvation`.

I don't understand your point about weirdness.

Reply:
ok then!